### PR TITLE
Add Integration test for hpc-slurm-chromedesktop

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  region: europe-west1
+  zone: "{{ zone }}"
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/community/examples/hpc-slurm-chromedesktop.yaml"
+network: "{{ deployment_name }}-net"
+max_nodes: 5
+# Note: Pattern matching in gcloud only supports 1 wildcard.
+login_node: "{{ slurm_cluster_name }}-login-*"
+controller_node: "{{ slurm_cluster_name }}-controller"
+post_deploy_tests: []
```
Comment (mr0re1) [inline tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml:32]: Please add partitions and mounts to check

Comment (harshthakkar01) [reply tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml]: Added, mount and partition test. It seems we had set dynamic node in the blueprint as 1 so partition test timed out. So, also updated that.

**Comment by mr0re1** [comment]
Pleaser address offline comments from Carlos, and assign back to me once it's ready for review.

**Review by mr0re1 — APPROVED** [review]
LGTM. Did you run this test?

**Comment by harshthakkar01** [comment]
Yes, I put the test results in the canonical bug.

**Review by cboneti — APPROVED** [review]

**Review by cboneti — APPROVED** [review]

**Pull request merged**